pkg/agent/openflow/pipeline.go

```diff
@@ -2632,7 +2736,7 @@ func (f *featureMulticast) externalMulticastReceiverFlow() binding.Flow {
 		MatchDstIPNet(*mcastCIDR).
 		Action().LoadRegMark(OFPortFoundRegMark).
 		Action().LoadToRegField(TargetOFPortField, config.HostGatewayOFPort).
-		Action().NextTable().
+		Action().GotoTable(MulticastOutputTable.GetID()).
```

Comment:
Prefer to use `GotoStage` if just wants to skip Ingress stage.

Reply:
done